### PR TITLE
Metehan

### PR DESCRIPTION
**Pull request by 4745Mthn**

### CODE REVIEW

**Review by dilovar1992 — APPROVED** [review]

**Review by dilovar1992 — APPROVED** [review]

File: src/test/resources/features/US08_CreateCalendarEvent_MA.feature [hunk 1]
```diff
@@ -7,9 +7,8 @@ Feature: Users should be able to login
   @Ac1
```
Comment (ersuamine) [inline src/test/resources/features/US08_CreateCalendarEvent_MA.feature:7]: Can you change feature description "Feature: Users should be able to login" on line 2
It should be describing your feature

File: src/test/resources/features/US01_DashboardModules_MA.feature [hunk 2]
```diff
@@ -1,13 +1,13 @@
-@login @regression @US01
+@US01 @regression
 Feature: Users should be able to login
```
Comment (ersuamine) [inline src/test/resources/features/US01_DashboardModules_MA.feature:2]: Please revise with more appropriate feature description

File: src/test/java/com/fleet_management/step_definitions/US08_CreateCalendarEvent_StepDef_MA.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Then("user cleared the Calendar event Repeat Every field")
+    public void userClearedTheCalendarEventRepeatEveryField() {
+        calendarEventsPage.repeatEveryDaysValue.clear();
+    }
+
+    @Then("user should be able to see {string} error message")
+
+    public void userShouldBeAbleToSeeErrorMessage(String expectedWarningMessage) {
+        Assert.assertTrue(expectedWarningMessage.contains(calendarEventsPage.warningMessage.getText()));
```
Comment (ersuamine) [inline src/test/java/com/fleet_management/step_definitions/US08_CreateCalendarEvent_StepDef_MA.java:44]: Should expected contain the actual, or the actual should contain the expected

**Review by ersuamine — CHANGES_REQUESTED** [review]

**Review by ersuamine — APPROVED** [review]
Thank you for your hard work

**Pull request merged**